packages/editor/src/core/types/editor.ts

```
@@ -1,20 +1,68 @@
+  "background-color": {
+    color: string | undefined;
+  };
+  "text-align": {
+    alignment: string;
+  };
+};
+
+// Create a utility type that maps a command to its extra props or an empty object if none are defined
+export type TCommandWithProps<T extends TEditorCommands> = T extends keyof TCommandExtraProps
+  ? TCommandExtraProps[T] // If the command has extra props, include them
+  : {}; // Otherwise, just return the command type with no extra props
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)